ml-package-versions-pr.yml

```diff
@@ -0,0 +1,42 @@
+sklearn:
+  models:
+    package: "scikit-learn"
+    minimum: "0.19.2"
+    maximum: "0.23.2"
+    run: |
+      pytest tests/sklearn/test_sklearn_model_export.py --large
+
+  autologging:
+    package: "scikit-learn"
+    minimum: "0.22.2"
+    maximum: "0.23.2"
+    unsupported: ["0.22.2"]
+    requirements: ["matplotlib"]
+    run: |
+      pytest tests/sklearn/test_sklearn_autolog.py --large
+
+  dev:
+    package: "scikit-learn"
+    install: |
+      pip install git+https://github.com/scikit-learn/scikit-learn.git@master
```

Comment:
Awesome that we're testing dev / nightly versions. Can we find nightlies and configure them for other libraries as well? For example: https://pypi.org/project/tf-nightly/ for TensorFlow

Reply:
- LightGBM: can be installed from master (requires build)
- XGBoost: can be istalled from master (requires build)
- pytorch: nightly version is available
- sklearn: can be installed from github repo.